packages/esm-patient-test-results-app/src/print/print.component.tsx

```diff
@@ -0,0 +1,257 @@
+          const formattedPanelDate = panelDate.toLocaleDateString();
+          return {
+            id: panel.id,
+            testType: panel.name,
+            date: formattedPanelDate,
+            result: panel.value,
+            normalRange: panel?.meta?.range ?? '--',
+          };
+        });
+    }
+
+    return panels.map((panel) => {
+      const panelDate = new Date(panel.effectiveDateTime);
+      const formattedPanelDate = panelDate.toLocaleDateString();
+      return {
+        id: panel.id,
+        testType: panel.name,
+        date: formattedPanelDate,
+        result: panel.value,
+        normalRange: panel?.meta?.range ?? '--',
+      };
+    });
+  }, [panels, selectedFromDate, selectedToDate]);
+
+  return (
+    <div>
+      <ModalBody>
+        <div style={{ display: 'flex', justifyContent: 'center' }}>
+          <div style={{ display: 'flex', alignItems: 'center' }}>
+            <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+              <div style={{ display: 'flex', alignItems: 'center' }}>
+                <span style={{ marginRight: '0.5rem' }}> From:</span>
+                <DatePicker
+                  id="onsetDate"
+                  datePickerType="single"
+                  dateFormat="d/m/Y"
+                  light={isTablet}
+                  onChange={fromDate}
+                  value={selectedFromDate}
+                >
+                  <DatePickerInput
+                    id="onsetDateInput"
+                    style={{
+                      width: '100%',
+                      marginRight: '0.5rem',
+                      border: '1px solid #ccc',
+                      padding: '0.5rem',
+                      fontSize: 'smaller',
+                    }}
+                  />
+                </DatePicker>
+              </div>
+              <div style={{ display: 'flex', alignItems: 'center' }}>
+                <span style={{ marginRight: '0.5rem' }}>To:</span>
+                <DatePicker
+                  id="onsetDate"
+                  datePickerType="single"
+                  dateFormat="d/m/Y"
+                  light={isTablet}
+                  onChange={toDate}
+                >
+                  <DatePickerInput
+                    id="onsetDateInput"
+                    style={{
+                      width: '100%',
+                      marginRight: '0.5rem',
+                      border: '1px solid #ccc',
+                      padding: '0.5rem',
+                      fontSize: 'smaller',
+                    }}
+                  />
+                </DatePicker>
+              </div>
+              <Button kind="secondary" size="sm" onClick={handlePrint}>
+                {t('print', 'Print')}
+              </Button>
+            </div>
+          </div>
+        </div>
+        <div ref={componentRef} className={styles.printpage}>
+          <div className={styles.printPage}>
```

Comment:
Which one is it? `printPage` or `printpage`?

Reply:
updated